src/org/usfirst/frc/team467/robot/DriverStation.java

```diff
@@ -89,11 +93,25 @@ public boolean getGyroReset() {
 	
 	public double getArcadeSpeed() {
 		return getDriveJoystick().turboSpeedAdjust();
-		
 	}
 	
 	public double getArcadeTurn() {
 		return getDriveJoystick().getRightStickX();
 	}
-
+	
+	public void setLeftRumble(double value) {
+		getDriveJoystick().setRumble(RumbleType.kLeftRumble, value);
+	}
+	
+	public void setRightRumble(double value) {
+		getDriveJoystick().setRumble(RumbleType.kRightRumble, value);
+	}
```

Comment:
Let's create a method "setRumble(double value)" that sets both sides at the same time.

Or, maybe you should have setDriverRumble() and setNavigatorRumble() methods.

Reply:
Call it setDriverRumble(), so when we add a navigator controller we can name it setNavigatorRumble().